### ISSUE TITLE
Update PoS parameter values

### ISSUE THREAD

**Issue by brentstone** [anoma/namada#478]
During the August retreat in Italy, we decided upon some parameter values for the PoS system. These are:

- pipeline_offset = 2
- unbonding_offset = 21
- slash rate = 0.001 / namnam (fundamental token unit)
- votes per token = 1 / namnam (fundamental token unit)
- max active validators = 100
- block proposer reward = 1% minimum
- PoS inflation = 10% maximum

Furthermore, subsequent conversations established:
- unit denomination = 10^6 (fundamental tokens per whole NAM)

**Comment by brentstone** [comment]
@cwgoes @adrianbrink @tzemanovic - do we want the same slash rate for duplicate vote attacks and light client attacks?

**Comment by cwgoes** [comment]
> @cwgoes @adrianbrink @tzemanovic - do we want the same slash rate for duplicate vote attacks and light client attacks?

Yes - both varying from 0 to 1 (100%) according to the cubic slashing function.
